_posts/2017-06-09-the-lead-developer-conference.md

```diff
@@ -411,7 +411,22 @@ We try to do this last one by organizing _Architecture sessions_, where we revie
 
 
 ### Carly Robinson - Mentoring Junior Engineers at Slack HQ <a name="#Carly_Robinson">&nbsp;</a>
-
+Up until a year and a half ago, Carly Robinson was a musical theatre actress, until she decided she wanted to make a huge career switch and become a software engineer. In this talk she tells her story of how and why she was able to achieve this goal at Slack HQ.
+
+#### Imposter syndrome
+Coming from a family of artists, she had no idea she would one day discover a love for software engineering and a drive to learn the craft. Starting off with with several coding boot camps like codecademy and hackbright (a women-only software engineering school at San Francisco), she felt the imposter syndrome creeping in even before she even got started. We, as a tech industry, need a culture of empathy and inclusion in order to make sure no talent is wasted because of this.
+
+#### What Slack does right
+* Mentorship is a relationship. Make sure it is a good fit from the beginning.
```

Comment:
Ik zou dit iets anders verwoorden: Evaluate the fit before it's too late (of iets in die aard). 'Make sure' is nogal vaag en sugereerd dat ze op voorhand beginnen zoeken naar specifieke kwaliteiten in een mentor.